scripts/run-integ-tests.ps1

```diff
@@ -19,7 +19,7 @@ Invoke-Expression "${PSScriptRoot}\..\misc\image-cleanup-test-images\build.ps1"
 $cwd = (pwd).Path
 try {
   cd "${PSScriptRoot}"
-  go test -tags integration -timeout=10m -v ../agent/engine ../agent/stats
+  go test -tags integration -timeout=15m -v ../agent/engine ../agent/stats
```

Comment:
I had a chat with @adnxn yesterday. The answer is maybe. I want to see if there is a way to run this test more deterministically.